lib/superstatic.js

```diff
@@ -19,6 +19,17 @@ var loadConfigFile = require('./loaders/config-file');
 
 var CWD = process.cwd();
 
+  // shink-ray is a fork of compression which supports advanced compression encodings but not ES5.
+function initializeCompressor(options) {
+  var compressor;
```

Comment:
You should move this `var` outside of the function, and then do `if (compressor) { return compressor; }` at the top. That way this would be memoized and it wouldn't be constantly retrying shrink-ray if it's not going to work.